modulus/models/layers/activations.py

```diff
@@ -95,3 +95,53 @@ def __init__(self):
 
     def forward(self, x: Tensor) -> Tensor:
         return 0.5 * (x + torch.sqrt(x * x + self.b))
+
+
+# Dictionary of activation functions
+ACT2FN = {
+    "relu": nn.ReLU(),
+    "leaky_relu": nn.LeakyReLU(0.1),
```

Comment:
Hmm, this is how they do it in HF https://github.com/huggingface/transformers/blob/main/src/transformers/activations.py#L213. Maybe we can have the leaky value as a tuple or something...

Reply:
Hmmm, yeah I was just thinking if someone wanted to try tuning things...

Not that big of a deal, since we ship complete models... users can always directly edit them for testing them. I think this is fine.